com.unity.render-pipelines.universal/Shaders/CameraMotionVectors.shader

```diff
@@ -54,15 +54,20 @@ Shader "Hidden/kMotion/CameraMotionVectors"
             // Fragment
             half4 frag(Varyings input, out float outDepth : SV_Depth) : SV_Target
             {
+                UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX(input);
                 // Calculate PositionInputs
                 half depth = LoadSceneDepth(input.position.xy).x;
                 outDepth = depth;
-                half2 screenSize = _ScreenSize.zw;
-                PositionInputs positionInputs = GetPositionInput(input.position.xy, screenSize, depth, UNITY_MATRIX_I_VP, UNITY_MATRIX_V);
 
-                // Calculate positions
-                float4 previousPositionVP = mul(_PrevViewProjM, float4(positionInputs.positionWS, 1.0));
-                float4 positionVP = mul(UNITY_MATRIX_VP, float4(positionInputs.positionWS, 1.0));
+                // Transform to world space
+                float2 ndcpos = input.position.xy *= _ScreenSize.zw;
+                float4 cspos = ComputeClipSpacePosition(ndcpos, depth);
```

Comment:
depth needs to be reversed for OGL platform. You can test this code work locally in OGL in the editor by switching the editor to OpenGL. There's a documentation page on how to reconstruct depth in URP that works on all platforms. https://docs.unity3d.com/Packages/com.unity.render-pipelines.universal@12.0/manual/writing-shaders-urp-reconstruct-world-position.html